no-context-help-links.yaml

```diff
@@ -0,0 +1,77 @@
+"appmanager/users":

```

Comment:
move this file to a separate branch that we will merge when content is available